readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -81,22 +76,6 @@ def append_conf(self, **kwargs):
                 '%scss/readthedocs-doc-embed.css' % MEDIA_URL,
             ]
 
-        if 'pages' not in user_config:
-            user_config['pages'] = []
-            for root, dirnames, filenames in os.walk(docs_dir):
-                for filename in filenames:
-                    if fnmatch.fnmatch(filename, '*.md') or fnmatch.fnmatch(filename, '*.markdown'):
-                        if docs_dir != '.':
-                            root_path = root.replace(docs_dir, '')
-                            full_path = os.path.join(root_path, filename.lstrip('/')).lstrip('/')
-                        else:
-                            if root == '.':
-                                root_path = ''
-                            else:
-                                root_path = re.sub('^./', '', root)
-                            full_path = os.path.join(root_path, filename.lstrip('/')).lstrip('/')
-                        user_config['pages'].append([full_path])
-
```

Comment:
MkDocs automatically includes all Markdown files now if the user doesn't manually list them. So this is just duplication.